### DISCUSSION TITLE
功能建议：增加播放设备选择

### DISCUSSION THREAD

**Discussion by Nekozuka-Hibiki** [SPlayer-Dev/SPlayer#243] [想法 Ideas]
功能建议：
提供播放输出设备的选择选项
 详：
三个播放设备
内置音箱
外设音箱
耳机
如果按照Windows默认状态不做改动，就是系统使用的默认设备为输出，方便点说就是Splayer提供选项，选择软件输出设备
例：
默认外设音箱播放，不与耳机串音
参考：
[YesPlayMusic](https://github.com/qier222/YesPlayMusic)

**Comment by imsyy** [comment]
目前已实现音频输出切换，请等待新版本重构完成

![image](https://github.com/user-attachments/assets/3af8dd98-49d7-48a7-8cce-322443fe0c93)

**Reply by Nekozuka-Hibiki** [reply]
善！
